### PR TITLE
Converted the project to use Avenir by default

### PR DESCRIPTION
**Pull request by jimmynotjim**
Converted the project to use Avenir by default

## Removals

- Removed `webfont-regular` from styles where it's no longer needed
- Removed `short-desc` from templates where it's no longer needed

## Changes

- Converted full-width-sans to full-width-serif

## Testing

- Check all pages, everything should be Avenir except for the full width molecule on `/document-detail/`

## Review

- @duelj 
- @ajbush 
- @anselmbradford 
- @KimberlyMunoz 
- @sebworks 
- @schaferjh 

## Screenshots

__Only serif (Georgia) in use__

screen shot 2016-01-21 at 5 50 48 pm

## Todos

- Move this over to CF at some point.

## Checklist

* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices 
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

**Comment by KimberlyMunoz** [comment]
http://localhost:8000/sub-pages/privacy-impact-assessments-pias/ has too much of the `.webfont-medium().`

image

File: cfgov/jinja2/v1/_includes/templates/nav/about-us-media.html [hunk 1]
```diff
@@ -1,11 +1,11 @@
 <a href="/the-bureau/">
     <img src="/static/img/nav-about-us-video.png"
          alt="About Us video image" />
-    <p class="h4 short-desc u-mt15">
+    <p class="h4 u-mt15">
       The CFPB: Working for you.
     </p>
 </a>
-<div class="short-desc">
```
Comment (KimberlyMunoz) [inline cfgov/jinja2/v1/_includes/templates/nav/about-us-media.html]: :+1: Good catch!

**Comment by jimmynotjim** [comment]
Fixed the Avenir Medium issue.

**Comment by KimberlyMunoz** [comment]
Checked all the pages and it looks good to me. :+1:

**Comment by anselmbradford** [comment]
screen shot 2016-01-22 at 1 29 29 pm

Looks like `We work with …` needs Avenir too, when compared to refresh anyway.

**Comment by anselmbradford** [comment]
:+1:

**Pull request merged**